pkg/issuer/issuer.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
 )
 
+ const (
+ 	// Error reason generated when duration or renewBefore is invalid
+	ErrorDurationInvalid = "ErrDurationInvalid"
```

Comment:
A lot of things to do with constants seemed to move since the original commit, so I stuck this here, Is there a better place for it?

Reply:
From what I can tell, this var isn't actually used? Unless GitHub is collapsing things in the diff for me! 😄

Reply:
Yup you're right, this is a holdover from when we were triggering events for stuff (I think) so I'll remove it